facebook-commerce.php

```diff
@@ -1527,6 +1533,7 @@ public function create_or_update_product_set_item( $product_set_data, $product_s
 				);
 			}
 		} catch ( ApiException $e ) {
+
 			$message = sprintf( 'There was an error trying to create/update a product set: %s', $e->getMessage() );
```

Comment:
This along with all the messages displayed to the user will need to be made translatable